BraveRewardsUI/Extensions/DateExtensions.swift

```diff
@@ -27,4 +27,13 @@ extension Date {
     let calendar = Calendar(identifier: .gregorian)
     return calendar.component(.month, from: self)
   }
+  
+  static func dateStringFrom(_ intervalSince1970: UInt64) -> String {
```

Comment:
API naming: `dateStringFrom(_:)` → `dateString(from:)` or `dateStringFrom(intervalSince1970:)`

Reply:
Required tests: Other method in this extension are tested in [`DateExtensionTests`](https://github.com/brave/brave-rewards-ios/blob/master/BraveRewardsTests/DateExtensionsTests.swift).  Should add a test here to test this method

Reply:
Also why is `intervalSince1970` a `UInt64` instead of a `TimeInterval`?

Reply:
The `reconcileStamp` is `UInt64`, so to use this method we have to cast everytime. And since this method was created for conversion of `reconcileStamp` to string, I kept the param as UInt64

Reply:
I realize that but nothing in the name of this specifies it's for reconcile stamp. 

The way it's written (name, location, etc) all just make this a convenience method for using a date formatted from a Unix timestamp.

Reply:
updated the param to `reconcileStamp`

Reply:
Sorry, one more nit: I just realized this is an extension on `Date`, so prefixing with `date` in the method is redundant. Can just be `string(from:)` or `stringFrom(reconcileStamp:)`